### PR TITLE
fix missing of resource command & cgroup manager

### PR DESCRIPTION
**Pull request by Mr-Linus**

### CODE REVIEW

**Comment by Mr-Linus** [comment]
I have fixed missing of resource command & cgroup manager in [code-4.1](https://github.com/Mr-Linus/mydocker/tree/code-4.1). But Github does not support pulling request the tag to another tag.
Could you create a new branch based in tag code-4.1 so that I can pull request ?

**Comment by xianlubird** [comment]
Done  @Mr-Linus  you can push branch code-4.1

**Comment by Mr-Linus** [comment]
Complete switching branch.
@xianlubird

File: main_command.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/urfave/cli"
 	"github.com/xianlubird/mydocker/container"
+	"./cgroups/subsystems"
```
Comment (xianlubird) [inline main_command.go]: 不要用相对路径  改为github开头的绝对路径

File: run.go [hunk 2]
```diff
@@ -5,10 +5,12 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"os"
 	"strings"
+	"./cgroups/subsystems"
```
Comment (xianlubird) [inline run.go]: 一样

**Comment by Mr-Linus** [comment]
已修正 @xianlubird

**Pull request merged**